src/styles/blocks/card.scss

```diff
@@ -0,0 +1,76 @@
+@use '../_variables/variables' as *;
+
+.card {
+  width: $CardWidth;
+  padding: $CardWidth * 0.01 * 16 $CardWidth * 0.01 * 8 $CardWidth * 0.01 * 8;
+  border: 1px solid #f3f3f3;
+
+  &__image {
+    width: $CardWidth * 0.8;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: $FontMedium;
```

Comment:
The variable `$FontMedium` should be defined in the `variables.scss` file. Ensure that it is correctly defined and imported.